doc/release/yarp_3_4/detect_clock_reset.md

```diff
@@ -0,0 +1,8 @@
+detect_clock_reset {#yarp_3_4}
+-----------------------
+
+### Libraries
+
+#### network clock
```

Comment:
```suggestion
#### `os` 
```
The library in which the NetworkClock is `libYARP_os`, so this section should indicate that (see the other files in the directory.